README.md

```diff
@@ -62,4 +62,35 @@ Validator validator = new FastaValidator(isr, SequenceType.DNA);
 ValidationResult result = validator.validate();
 
 assertTrue(result.isValid());
-```
\ No newline at end of file
+```
+
+### Using biovalidator in your application
+
+Maven dependency
+```xml
+<dependency>
+  <groupId>org.intermine</groupId>
+  <artifactId>biovalidator</artifactId>
+  <version>0.1.0</version>
+  <type>pom</type>
+</dependency>
+```
+
+Gradle dependency
+```groovy
+compile 'org.intermine:biovalidator:0.1.0'
+```
+
+### Using biovalidator as a command line utility
+Step 1: Get biovalidator Fat-Jar(either build from source or dirctly download from [bintray](https://bintray.com/intermineorg/biovalidator/biovalidator#files/org%2Fintermine%2Fbiovalidator%2F0.1.0))
```

Comment:
1. you have bintray listed twice. remove one link
2. you don't mention the generated jar will be in `build/libs`. please add.